.github/workflows/test.yml

```diff
@@ -149,7 +149,7 @@ jobs:
       # the old gaiad binary version is hardcoded, need to be updated each major release.
       - name: Install Old Gaiad
         run: |
-          git checkout v11.0.0 
+          git checkout v12.0.0-rc0
```

Comment:
Shouldn't this be v12.0.0?

Reply:
v12.0.0 is not released yet

Reply:
can wait v12.0.0 release, and then change here then merge PR.

Reply:
@yaruwangway https://github.com/cosmos/gaia/releases/tag/v12.0.0